### PR TITLE
[stable28] fix(admin): Fix caldav instructions for occ config:app:set

### PR DESCRIPTION
**Pull request by ChristophWurst**
### ☑️ Resolves

* Ref https://github.com/nextcloud/server/issues/44116#issuecomment-1987964522
* Backports https://github.com/nextcloud/documentation/pull/11636

### 🖼️ Screenshots

Bildschirmfoto vom 2024-03-11 10-32-02

### CODE REVIEW

**Comment by ChristophWurst** [comment]
@JB1985 @gramels please review

File: admin_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -148,15 +148,15 @@ Rate limits
 Nextcloud rate limits the creation of calendars and subscriptions if too many items are created within a short time frame. The default is 10 calendars or subscriptions per hour. This can be customized as follows::
 
   # Set limit to 15 items per 30 minutes
-  php occ config:app:set calendar rateLimitCalendarCreation --type=integer --value=15
-  php occ config:app:set calendar rateLimitPeriodCalendarCreation --type=integer --value=1800
+  php occ config:app:set calendar rateLimitCalendarCreation --value=15
+  php occ config:app:set calendar rateLimitPeriodCalendarCreation --value=1800
```
Comment (tcitworld) [inline admin_manual/groupware/calendar.rst:152]:
```suggestion
  php occ config:app:set dav rateLimitCalendarCreation --value=15
  php occ config:app:set dav rateLimitPeriodCalendarCreation --value=1800
```

File: admin_manual/groupware/calendar.rst [hunk 2]
```diff
@@ -148,15 +148,15 @@ Rate limits
...
 
   # Allow users to create 50 calendars/subscriptions
-  php occ config:app:set calendar maximumCalendarsSubscriptions --type=integer --value=50
+  php occ config:app:set calendar maximumCalendarsSubscriptions --value=50
 
 or::
 
   # Allow users to create calendars/subscriptions without restriction
-  php occ config:app:set calendar maximumCalendarsSubscriptions --type=integer --value=-1
+  php occ config:app:set calendar maximumCalendarsSubscriptions --value=-1
```
Comment (tcitworld) [inline admin_manual/groupware/calendar.rst:162]:
```suggestion
  php occ config:app:set dav maximumCalendarsSubscriptions --value=-1
```

File: admin_manual/groupware/calendar.rst [hunk 3]
```diff
@@ -148,15 +148,15 @@ Rate limits
...
-  php occ config:app:set calendar rateLimitCalendarCreation --type=integer --value=15
-  php occ config:app:set calendar rateLimitPeriodCalendarCreation --type=integer --value=1800
+  php occ config:app:set calendar rateLimitCalendarCreation --value=15
+  php occ config:app:set calendar rateLimitPeriodCalendarCreation --value=1800
 
 Additionally, the maximum number of calendars and subscriptions a user may create is limited to 30 items. This can be customized too::
 
   # Allow users to create 50 calendars/subscriptions
-  php occ config:app:set calendar maximumCalendarsSubscriptions --type=integer --value=50
+  php occ config:app:set calendar maximumCalendarsSubscriptions --value=50
```
Comment (tcitworld) [inline admin_manual/groupware/calendar.rst:157]:
```suggestion
  php occ config:app:set dav maximumCalendarsSubscriptions --value=50
```

File: admin_manual/groupware/calendar.rst (hunk 1 above)
Comment (ChristophWurst) [reply admin_manual/groupware/calendar.rst:152]: Yep. I'll cherry-pick https://github.com/nextcloud/documentation/pull/11636 onto this branch

**Review by miaulalala — APPROVED** [review]

**Pull request merged**